registry/storage/driver/s3-aws/s3.go

```diff
@@ -667,6 +670,7 @@ func (d *driver) logS3Operation(ctx context.Context) request.NamedHandler {
 				"s3_http_response_header_x-amz-request-id": resp.Header.Values("x-amz-request-id"),
 				"s3_http_response_status":                  resp.StatusCode,
 				"s3_http_response_content-length":          resp.ContentLength,
+				"s3_http_request_duration":                 duration.Seconds(),
```

Comment:
What do you think of including both `request.Request.RetryCount` and `request.Request.AttemptTime` as you did in https://github.internal.digitalocean.com/digitalocean/docr/pull/99?

Not suggesting `request.Request.AttemptTime` as an alternative to the duration you've already added, but in addition to -- just in case they are somehow different (eg `time.Now().Sub(r.Time)` could include time spent blocked waiting on sockets or something whereas `request.Request.AttemptTime` may not).

Reply:
Sure I can add these